src/CMakeLists.txt

```diff
@@ -110,6 +108,7 @@ install(
 
 install(EXPORT chai-targets DESTINATION share/chai/cmake/)
 
+
```

Comment:
What's this for?

Reply:
Errr.  Uhhhh. Just decided to insert a random line break??  I’ll remove it.